### ISSUE TITLE
[bug]: Heading structure invalid in documentation

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5252]
### 🐞 Describe the bug

Various pages have invalid heading structure:
- [Packages > Internet-header](https://next.design-system.post.ch/?path=/docs/cfa21cbc-5f8d-4a42-8165-7f6ed20bf133--docs): `Request a configuration` is an `h5` that comes right after an `h1`, which is invalid. It should be an `h2` and the headings under (Step 1 and Step 2) should therefore be `h3` elements.
- [Foundations > Typography > Lists](https://next.design-system.post.ch/?path=/docs/e76192bb-b2eb-487a-b9c1-ef938bccdfc4--docs): `Sass mixin` is an `h4` directly placed after an `h2` element. The error occurs twice in the page.
- [Foundations > Layout > Grid](https://next.design-system.post.ch/?path=/docs/7240f2ef-216a-490e-9bd8-c0cef19f7b31--docs): There is an `h4` in the banner, directly following an `h2`.
- [Foundations > Layout > Columns](https://next.design-system.post.ch/?path=/docs/cd70f48f-6308-4ec8-a8c6-982bd72c855c--docs): The title "Alignment" is an `h3` directly following an `h2`.
- [Components > Datepicker](https://next.design-system.post.ch/?path=/docs/6561b345-9d64-4f95-8ea6-2389586410bd--docs): The banner contains an `h4` directly after an `h1` element.
- [Components > Dropdown](https://next.design-system.post.ch/?path=/docs/63113d0a-6ceb-472c-ac6a-729404e862ca--docs): The banner contains an `h4` directly after an `h1` element.
- [Components > Pagination](https://next.design-system.post.ch/?path=/docs/caa89695-5c39-4cb5-8e62-579d918cbd2f--docs): The banner contains an `h4` directly after an `h1` element.
- [Components > Progressbar](https://next.design-system.post.ch/?path=/docs/446648d9-0360-4dec-a6f4-87f2f28b5f33--docs): The banner contains an `h4` directly after an `h1` element.
- [Components > Timepicker](https://next.design-system.post.ch/?path=/docs/a4d9f996-6e77-4fad-bb19-40fad1bb7a1d--docs): The banner contains an `h4` directly after an `h1` element.
- [Components > Typeahead](https://next.design-system.post.ch/?path=/docs/0456901c-3713-4f1f-b592-1f4dbd65a500--docs): The banner contains an `h4` directly after an `h1` element.
- [Utilities > Display](https://next.design-system.post.ch/?path=/docs/1bb8a4a7-8205-4dcb-a41c-8f7ab1de1c99--docs): `Available display values` and `Usage` should be `h2` elements and not `h3`.
- [Utilities > Opacity](https://next.design-system.post.ch/?path=/docs/5e27e48d-a5f6-4e57-a343-7f40507fc27b--docs): `Example` should be an `h2`.
- [Utilities > Spacing](https://next.design-system.post.ch/?path=/docs/facaacfd-18f1-49b4-80f1-a96680730fa0--docs): Banner has an `h4` element after an `h2` element.
- [Guidelines > Styling shadowdom parts](https://next.design-system.post.ch/?path=/docs/ba379b2d-689f-4003-8f6f-011af341549b--docs): `Example` is an `h3` element just after the `h1` element.
